project/F84064014/README.rst

```diff
@@ -0,0 +1,77 @@
+======================================================================
+A Simple End-to-End NN Framework for Deep Learning
+======================================================================
+
+Basic Information
+=================
+
+Deep Learning is now widely used to solve a lot of problem from many different field, 
+such as computer vision, data classification, natural language processing, etc., this
+project we will implement a End-to-End Deep Learning Framework and accelerate it with C++
+
+Problem to Solve
+================
+
+In the project we try to make deep learning faster and easier
+
+Deep Learning is a powerful tool even for people who never write code before, however, 
+build a deep learning model is not a walk in the park, we hope to build a framework
+that user can simply use deep learning model by only put data in and get data out.
+
+Neaural Network is a important part of deep learning model. Training a neural network
+may cost a lot of time especially when there is no gpu available, thus some component
+would be written in C++ to make it faster
+
+Prospective Users
+=================
+
+People who wants to use deep learning method but have no idea of programming or people
+who just start to learn about deep learning.
+
+System Architecture
+===================
+
+All function and class will be defined in python class FNN, function with complicate 
+computation process like forward, backward propagation, etc., would be written in C++
```

Comment:
Similarly, I think you may use a list for functionalities that will be done in C++.